basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/SocketBindingsTestCase.java

```diff
@@ -23,6 +23,9 @@
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 import org.openqa.selenium.WebDriver;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+
+import java.io.IOException;
```

Comment:
You removed throwing these exceptions so you have to remove now unused imports of both these exceptions as well.